bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1596,7 +1620,11 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
          ic_ymd="19790101"  sim_year="1979" do_transient_pfts=".false." use_excess_ice=".false."
          ic_tod="0" glc_nec="10" use_crop=".false." irrigate=".true."
          lnd_tuning_mode="clm6_0_cam7.0" use_init_interp=".true."
->lnd/clm2/initdata_map/clmi.BHIST.2000-01-01.0.9x1.25_gx1v7_simyr1979_c200806.nc
+>lnd/clm2/initdata_map/ctsm53n04ctsm52028_f09_hist.clm2.r.1979-01-01-00000.nc
+</finidat>
+<finidat hgrid="ne30np4.pg3"  mask="t232" sim_year="1979"
```

Comment:
More attributes as above.

Reply:
Added but does not work, yet.

Reply:
Discussion about this one continues elsewhere, so I'm resolving this one.